apps/transport/lib/transport_web/controllers/validation_controller.ex

```diff
@@ -150,7 +162,7 @@ defmodule TransportWeb.ValidationController do
   defp filepath(type) do
     cond do
       type == "tableschema" -> Ecto.UUID.generate() <> ".csv"
-      type in ["jsonschema", "gtfs"] -> Ecto.UUID.generate()
+      type in ["jsonschema", "gtfs", "netex"] -> Ecto.UUID.generate()
```

Comment:
```suggestion
      true -> Ecto.UUID.generate()
```

à envisager ? Si seulement 2 clauses peut-être juste `if/else`

Reply:
Hmmm, en ajoutant "netex" dans le deuxième cas les 2 premières assertions sont exhaustives ?

Reply:
Fait.